app/mailers/user_mailer.rb

```diff
@@ -89,6 +89,21 @@ def work_package_watcher_added(work_package, user, watcher_setter)
     end
   end
 
+  def work_package_watcher_removed(work_package, user, watcher_remover)
+    User.execute_as user do
```

Comment:
Let's eliminate blocks and arguments with brackets `User.execute_as(user) do`

Reply:
That's the style of the project.